hard/ultrascale_plus/bsg_link/tcl/bsg_link_ddr.sample_constraints.tcl

```diff
@@ -0,0 +1,40 @@
+# User should re-define periods / names / ports / pins / margins
+# Then add the following lines into the .xdc constraint file
+
```

Comment:
Do TCL functions still work in .xdc? If so, can we wrap this in a function so the user can pass in variables instead of copy paste?

Reply:
All XDC constraints can be applied by sourcing a .tcl script with the same commands.  Avoid using "read_xdc" with constraints spec'd in a tcl file as this will overwrite the file type property back to .xdc.

Reply:
TCL version added.